src/state/translations-mixin.ts

```diff
@@ -77,7 +75,15 @@ export default <T extends Constructor<HassBaseEl>>(superClass: T) =>
     private __loadedTranslations: {
       // track what things have been loaded
       [category: string]: LoadedTranslationCategory;
-    } = {};
+    } = {
+      // state is no longer fetched from the backend but it is
+      // still used in the frontend so we need to initialize it
+      state: {
+        integrations: [],
+        setup: false,
+        configFlow: false,
+      },
```

Comment:
Is this added as a precaution or are there specific cases where frontend still uses this?

Reply:
The UI is pretty broken without this.

Reply:
Would you mind giving an example? I've applied your code changes with and without this part and UI remains to load in both cases. It sounds like there still somewhere a fallback to state translations that may be applied.

Reply:
I removed it, and it seems fine today.  Yesterday all the translations wouldn't load without it. Perhaps it was a cache problem or something dirty on my local branch.

I've cleared the cache and tried 3 different browsers and it still works so I'm not sure what has changed between yesterday and today, but happy to not have to have the code if its not needed